### PR TITLE
make /rand inclusive + add /rn for vanilla clients

### PR DESCRIPTION
**Pull request by MargaretTheFool**
i lied and said it was inclusive in the string for it but it wasnt

### CODE REVIEW

**Comment by MargaretTheFool** [comment]
ok i DONT know why my commits for something else went into this too

**Comment by 0xDrMoe** [comment]
is this ready if /rn is here?

File: Patches/ChatCommandPatch.cs [hunk 1]
```diff
@@ -1482,6 +1482,16 @@ public static void OnReceiveChat(PlayerControl player, string text, out bool can
...
+            case "/rn":
+            case "/rename":
+            case "/renomear":
+                if (args.Length < 1) break;
+                if (args.Skip(1).Join(delimiter: " ").Length is > 10 or < 1) {
+                    Utils.SendMessage(GetString("Message.AllowNameLength"), player.PlayerId);
+                    break;
+                }
+                Main.AllPlayerNames[player.PlayerId] = args.Skip(1).Join(delimiter: " ");
+                break;
```
Comment (Tommy-XL) [inline Patches/ChatCommandPatch.cs:1494]: Wouldn't it be better to do this as a setting?

Comment (Tommy-XL) [reply Patches/ChatCommandPatch.cs:1494]: For example "Vanilla players can use command /rename"

**Comment by MargaretTheFool** [comment]
ok NOW everything should be ready

**Pull request merged**